assets/locales/core.en.toml

```
@@ -388,3 +388,63 @@ one = "Hide"
 [HideThis]
 description = "Hide this"
 one = "Hide this"
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# FEEDBACK BAR
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+[Helpful]
+description = "Is this page useful?"
+one = "Is this page useful?"
+
+[Yes]
+description = "Yes"
+one = "Yes"
+
+[YesAria]
+description = "Yes I found this page useful"
+one = "Yes I found this page useful"
+
+[No]
+description = "No"
+one = "No"
+
+[NoAria]
+description = "No I didn't find this page useful"
+one = No I didn't find this page useful"
```

Missing opening double quote

```suggestion
one = "No I didn't find this page useful"
```

---

good spot. fixed!